### DISCUSSION TITLE
PWA support/alternative?

### DISCUSSION THREAD

**Discussion by Ketchupppp** [minbrowser/min#2024] [Ideas]
Im big noob when it comes to coding and everything associated with it so please be patient with me, haha.
I was was wondering if there is, will be, or if an alternative exists for PWA. My use case is youtube music. I just want it on the taskbar and for it to open its own contained window of just youtube music. the desktop app [https://github.com/ytmdesktop/ytmdesktop](url) is a resource hog but very rich and while the pwa app is light, im still running chrome on my pc and id like to avoid that.

Thanks!

**Comment by baboo35** [comment]
PWA running on Min engine would be great. It's like PWA with Chromium, but less resources used. Vote for this feature!
